starters/deno-oak-denodb/src/graphql/interfaces/codegen.ts

```diff
@@ -0,0 +1,192 @@
+import { GraphQLResolveInfo } from 'graphql';
+export type Maybe<T> = T | null;
+export type InputMaybe<T> = Maybe<T>;
+export type Exact<T extends { [key: string]: unknown }> = { [K in keyof T]: T[K] };
+export type MakeOptional<T, K extends keyof T> = Omit<T, K> & { [SubKey in K]?: Maybe<T[SubKey]> };
+export type MakeMaybe<T, K extends keyof T> = Omit<T, K> & { [SubKey in K]: Maybe<T[SubKey]> };
+/** All built-in and custom scalars, mapped to their actual values */
+export type Scalars = {
+  ID: string;
+  String: string;
+  Boolean: boolean;
+  Int: number;
+  Float: number;
+};
+
+export type Mutation = {
+  __typename?: 'Mutation';
+  createTechnology: Maybe<Technology>;
+  deleteTechnologyById: ResolveType;
+  updateTechnology: ResolveType;
+};
+
+
+export type MutationCreateTechnologyArgs = {
+  technology: InputMaybe<TechnologyInput>;
+};
+
+
+export type MutationDeleteTechnologyByIdArgs = {
+  id: InputMaybe<Scalars['String']>;
+};
+
+
+export type MutationUpdateTechnologyArgs = {
+  id: InputMaybe<Scalars['String']>;
+  input: InputMaybe<TechnologyInput>;
+};
+
+export type Query = {
+  __typename?: 'Query';
+  getTechnologies: Maybe<Array<Technology>>;
+  getTechnology: Maybe<Technology>;
+};
+
+
+export type QueryGetTechnologyArgs = {
+  id: InputMaybe<Scalars['String']>;
+};
+
+export type ResolveType = {
+  __typename?: 'ResolveType';
+  done: Maybe<Scalars['Boolean']>;
+};
+
+export type Technology = {
+  __typename?: 'Technology';
+  createdAt: Maybe<Scalars['String']>;
+  description: Maybe<Scalars['String']>;
+  displayName: Maybe<Scalars['String']>;
+  id: Maybe<Scalars['String']>;
+  updatedAt: Maybe<Scalars['String']>;
+  url: Maybe<Scalars['String']>;
+};
+
+export type TechnologyInput = {
```

Comment:
The types are generated here, but they don't seem to be used anywhere. Shouldn't they be used in other modules?